setup.cfg

```diff
@@ -1,6 +1,3 @@
-[bdist_wheel]
-universal=1
-
 [mypy]
```

Comment:
Could this be moved into pyproject.toml?

Reply:
Yes that's in another PR but I suppose this one is more appropriate.

Reply:
Whoops, it actually is already there :P https://github.com/anaconda/anaconda-client/pull/738/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R29-R30